components/MonogramAvatar/MonogramAvatar.jsx

```diff
@@ -7,25 +7,31 @@ import { isEmpty, map } from 'lodash'
 import styles from './monogram_avatar.scss'
 
 import Avatar from '../Avatar/Avatar'
+import AvatarSize from '../Avatar/Avatar'
```

Comment:
Is this working?
I think it should be `import type AvatarSize from '../Avatar/Avatar`

Reply:
It worked. I added the `type` keyword and it didn't make a difference. I would expect that declaring it as a type in `Avatar.jsx` would be enough.... but then again this is my first time working with Flow!